src/reference/chat/data.json

```diff
@@ -618,6 +618,45 @@
               "meta": {}
             }
           }
+        },
+        {
+          "title": "Censored Message",
+          "description": "A channel owner can configure their channel's <a href=\"https://aka.ms/mixercatbot\" target=\"_blank\">Catbot auto moderation</a> settings so that unwanted chat messages are automatically dropped. If a message is sent and that message violates this filter, only channel moderators and the channel owner will receive that message, with the `censored` meta property set as shown below. Other users shall not receive the chat message.",
```

Comment:
we should have the template render this as markdown

Reply:
The HTML here is to make sure the link opens in a new tab